<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frm_pdfDec</class>
 <widget class="QWidget" name="frm_pdfDec">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>841</width>
    <height>887</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>pdf entschlüsseln</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(113, 139, 139)</string>
  </property>
  <widget class="QLabel" name="lbl_img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>671</width>
     <height>841</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 244, 189)</string>
   </property>
   <property name="text">
    <string>Datei hierhin ziehen</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_datei">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>850</y>
     <width>671</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Sunken</enum>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_finden">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>850</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 202, 109)</string>
   </property>
   <property name="text">
    <string>pdf suchen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_ende">
   <property name="geometry">
    <rect>
     <x>694</x>
     <y>20</y>
     <width>131</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <stylestrategy>PreferAntialias</stylestrategy>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(190, 202, 109)</string>
   </property>
   <property name="text">
    <string>Ende</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_decrypted">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>410</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(200, 0, 0)</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::Panel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="text">
    <string>entschlüsselt</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
